### ISSUE TITLE
About use_sigmoid and softmax cross_entropy

### ISSUE THREAD

**Issue by Never-Walk-Away** [tztztztztz/eqlv2#20]
Hi, there is have a question about the the loss_cls of bbox_head of roi_head.
i think the final calssifier is a Multi classification. why choose sigmoid function instead of softmax function.

**Comment by tztztztztz** [comment]
I use the sigmoid loss function to model the detection problem as a set of independent tasks, that is: for each proposal, we estimated the probability of being each category, then output all those boxes.

In fact, even with softmax loss, instead of outputting a single box with the highest probability, we output <num_classes> boxes with the corresponding prob. 

If you just want to output a single box for a single proposal, you can choose the box with the highest probability, which you can do it in sigmoid as well.

**Comment by Never-Walk-Away** [comment]
Thank you for your reply! Best wishes for you~~~
